src/System.Security.Cryptography.Primitives/tests/CryptographicException.cs

```diff
@@ -20,5 +21,44 @@ public static void Ctor()
             Assert.Same(inner, new CryptographicException(message, inner).InnerException);
             Assert.Equal(message, new CryptographicException(message, inner).Message);
         }
+
+        [Theory]
+        [InlineData(0x00000000, "The operation completed successfully")]
+        [InlineData(0x80090013, "Invalid provider specified")]
+        [InlineData(0x80090016, "Keyset does not exist")]
```

Comment:
These could end up being different for different OS versions, right? Would we just wait for failure and tolerate different varieties?